### PR TITLE
release: PR for v0.7.2

### PR DESCRIPTION
**Pull request by systemsdt**
Updates in hyperion micro-service and charts

### CODE REVIEW

File: charts/devtron/devtron-bom.yaml [hunk 1]
```diff
@@ -200,3 +205,33 @@ monitoring:
...
+    imagePullPolicy: IfNotPresent
+    configs:
+      PG_ADDR: postgresql-postgresql.devtroncd
+      PG_DATABASE: casbin
+      PG_PORT: "5432"
+      PG_USER: postgres
+    dbconfig:
+      secretName: postgresql-postgresql
+      keyName: postgresql-password
+    resources:
```
Comment (badal773) [inline charts/devtron/devtron-bom.yaml:222]: Remove the resources like others microservice

File: charts/devtron/templates/casbin.yaml [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: casbin
+    release: devtron
+  name: casbin-service
+  namespace: devtroncd
+spec:
+  internalTrafficPolicy: Cluster
```
Comment (badal773) [inline charts/devtron/templates/casbin.yaml:109]: Only keep ports and selectors,type

File: charts/devtron/devtron-bom.yaml [hunk 3]
```diff
@@ -28,23 +28,28 @@ components:
         HIDE_EXCLUDE_INCLUDE_GIT_COMMITS: "false"
         ENABLE_SCOPED_VARIABLES: "true"
         ENABLE_CI_JOB: "true"
-        GLOBAL_API_TIMEOUT: "60000"
+        GLOBAL_API_TIMEOUT: "6000000"
```
Comment (badal773) [inline charts/devtron/devtron-bom.yaml:31]: why we increased the timeout

**Review by badal773 — DISMISSED** [review]

File: charts/devtron/devtron-bom.yaml (hunk 3 above)
Comment (akshatsinha007) [reply charts/devtron/devtron-bom.yaml]: I used the values which was specified on cd have reverted to default.

File: charts/devtron/devtron-bom.yaml [hunk 4]
```diff
@@ -30,21 +30,25 @@ components:
         ENABLE_CI_JOB: "true"
         GLOBAL_API_TIMEOUT: "60000"
         TRIGGER_API_TIMEOUT: "60000"
-        ENABLE_EXTERNAL_ARGO_CD: "false"
+        ENABLE_EXTERNAL_ARGO_CD: "true"
```
Comment (badal773) [inline charts/devtron/devtron-bom.yaml:33]: change the installer.release: "v0.7.2" in both values.yaml and bom.yaml

**Review by nishant-d — APPROVED** [review]

**Review by prakarsh-dt — APPROVED** [review]

**Pull request merged**